allauth_2fa/views.py

```diff
@@ -31,9 +32,19 @@ class TwoFactorAuthenticate(FormView):
     form_class = TOTPAuthenticateForm
     success_url = SUCCESS_URL
 
+    def get(self, request, *args, **kwargs):
```

Comment:
We probably want this shared logic in the `dispatch` method instead of overwriting `get`, `post`, etc.